src/PlaylistContainer/PlaylistContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const displayPlaylist = (playlistType: string) => {
+  if (playlistType === 'saved-playlist') {
+    return <h1>Saved Playlist</h1>
+  }
+  if (playlistType === 'generated-playlist') {
+    return <h1>Generated Playlist</h1>
+  }
+  if (playlistType === 'custom-playlist') {
+    return <h1>Custom Playlist</h1>
```

Comment:
This seems like a good way of avoiding conditional rendering without our `return`